src/communication/Publisher.cpp

```diff
@@ -10,3 +10,32 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "up-cpp/communication/Publisher.h"
+#include <up-cpp/datamodel/builder/UMessage.h>
+
+namespace uprotocol::communication {
+using namespace uprotocol::datamodel::builder;
+
+Publisher::Publisher(std::shared_ptr<transport::UTransport> transport,
+          v1::UUri& topic, v1::UPayloadFormat format,
```

Comment:
Topic should either remain const or change to &&. Since it is likely that topic URIs will be const at their source, a const reference is preferred here.

Reply:
done